includes/Core/Util/Activation_Notice.php

```diff
@@ -0,0 +1,163 @@
+<?php
+/**
+ * Class Google\Site_Kit\Core\Util\Activation_Notice
+ *
+ * @package   Google\Site_Kit
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Util;
+
+use Google\Site_Kit\Context;
+use Google\Site_Kit\Core\Admin\Notice;
+use Google\Site_Kit\Core\Assets\Assets;
+use Google\Site_Kit\Core\Util\Requires_Javascript_Trait;
+
+/**
+ * Class handling plugin activation.
+ *
+ * @since 1.0.0
+ * @since n.e.x.t Renamed from Activation.
```

Comment:
I feel like the `@since 1.0.0` doesn't really belong in this file, since it's a new class. The note about the class it came from in the "main" since annotation should be enough that "before this version it was handled in this other class". I'm not strongly against it, but it just seems odd that we would include a `@since` version on a class that didn't exist at the time.

Reply:
The other `@since 1.0.0` annotations in this file should be updated as well, no?

Reply:
Oh yeah, right. Done!